src/style/app.module.css

```diff
@@ -6390,3 +6487,582 @@ button[data-testid='createPostBtn'] {
   font-weight: bold;
   color: var(--black-color);
 }
+
+/* usersidebar  */
+
+.leftDrawer {
+  width: calc(300px);
+  background-color: var(--profile-bg) !important;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: 100;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 1rem 0 1rem;
+  background-color: var(--bs-white);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`